tests/subsys/pm/device_runtime_api/testcase.yaml

```diff
@@ -1,11 +1,14 @@
+common:
+  tags: pm
+  platform_allow:
+  - native_sim
+
 tests:
   pm.device_runtime.api:
-    tags: pm
     integration_platforms:
```

Comment:
Does repeating the `platform_allow` as `integration_platforms` do anything?

Reply:
not in this case that we have only one platform available AFAIU.

Have changed it.